src/utils/apiResponse.utils.ts

```diff
@@ -0,0 +1,29 @@
+/* eslint-disable @typescript-eslint/no-explicit-any */
+export type ControllerResponse = {
+  isSuccess: number;
+  statusCode: number;
+  message: string;
+  data?: any;
```

Comment:
PullProofAI

Consider using specific types instead of `any` for `data` and `error` fields in `ControllerResponse` to improve type safety.
```diff
  data?: unknown;
  error?: unknown;
```



<!-- This is an auto-generated comment by PullProofAI -->